data_model/src/query/sorting.rs

```diff
@@ -38,13 +38,13 @@ impl Sorting {
     }
 }
 
-impl From<Sorting> for Vec<(&'static str, Name)> {
-    fn from(sorting: Sorting) -> Self {
-        if let Some(key) = sorting.sort_by_metadata_key {
-            return vec![(SORT_BY_KEY, key)];
-        }
-
-        Vec::new()
+impl Sorting {
+    /// Converts self to Vec of tuples to be used in queries
+    ///
+    /// The length of the output Vec is not constant and it's in (0..3)
```

Comment:
It looks like here length of `Vec` can be either 0 or 1, not 2.

Reply:
Yep, going to change the description